js/src/ui/Form/AddressSelect/addressSelect.js

```diff
@@ -182,17 +183,20 @@ class AddressSelect extends Component {
         <label className={ styles.label } htmlFor={ id }>
           { label }
         </label>
-        <input
-          id={ id }
-          className={ styles.input }
-          placeholder={ ilHint }
+        <div className={ styles.outerInput }>
+          <input
+            id={ id }
+            className={ styles.input }
+            placeholder={ ilHint }
 
-          onBlur={ this.handleInputBlur }
-          onFocus={ this.handleInputFocus }
-          onChange={ this.handleChange }
+            onBlur={ this.handleInputBlur }
```

Comment:
Alphabetical order is fine, just aligning style everywhere, no extra breaks needed. (I would rather not introduce more styling preferences yet again, props is still all-over the place.)